packages/web3-core/test/unit/web3_request_manager.test.ts

```diff
@@ -280,6 +280,21 @@ describe('Web3RequestManager', () => {
 			expect(myProvider.request).toHaveBeenCalledTimes(1);
 			expect(await pr).toBe('test');
 		});
+
+		it('promise of legacy provider should be error', async () => {
```

Comment:
Sorry, cannot see something related to testing promise with `error` here.

Reply:
this was supposed to be removed, thanks